cell.go

```diff
@@ -397,8 +397,9 @@ func (f *File) SetCellStr(sheet, cell, value string) error {
 // setCellString provides a function to set string type to shared string
 // table.
 func (f *File) setCellString(value string) (t, v string, err error) {
-	if len(value) > TotalCellChars {
-		value = value[:TotalCellChars]
+	value_runes := []rune(value)
```

Comment:
Use camel case style name in Go variables. I agree with you, we can using the utf8 std library at here:

```go
if utf8.RuneCountInString(value) > TotalCellChars {
    value = string([]rune(value)[:TotalCellChars])
}
```

Reply:
Done

Reply:
Done